src/config.rs

```diff
@@ -0,0 +1,122 @@
+use clap::Parser;
+use num_rational::Ratio;
+use serde::Serialize;
+
+use crate::seat::Seat;
+
+#[derive(Parser, Serialize, Debug)]
+pub struct Config {
+    #[arg(long)]
+    pub num_blocks: u64,
+    #[arg(long)]
+    pub num_shards: u64,
+    /// TODO mention excess validators are ignored/unassigned.
+    #[arg(long)]
+    pub seats_per_shard: u64,
+    /// The amount of stake required to get one seat.
+    #[arg(long)]
+    pub stake_per_seat: u64,
```

Comment:
I wonder if all stake-related variables should actually be u128 since I think that is the data type used for balances in nearcore.

Reply:
True! I’ll change their type to `u128`.

Reply:
Changed in 32d9dbf.